Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Dockerfile
+
+# Use an official Ruby image with the desired Ruby version
+FROM ruby:3.1
+
+# Set build arguments for user ID and group ID
+ARG USER_ID
+ARG GROUP_ID
+
+# Create a group and user matching the host USER_ID and GROUP_ID to avoid permission issues
+RUN groupadd -g $GROUP_ID shopifygroup && \
```

Comment:
userid and groupid can be entirely dropped. we might not need a dedicated user within the container.

within the compose file, we should be able to use

```yaml
services:
  shopfiy:
    user: "${UID:-1000}:${GID:-1000}"
```

The UID and GID vars can be set in the `.env` file. It can automatically added via the `set-up.sh` script.

Reply:
checking this.

Reply:
@gowrizrh 

Thanks for the suggestion about removing the dedicated user and group setup in the Dockerfile and managing it through Docker Compose with user: "${UID:-1000}:${GID:-1000}". I really like the idea of simplifying the setup.

I did test this approach, but ran into some challenges since middleware like Shopify CLI and Node.js seem to require proper system user entries to resolve user information that isn’t available with just the USER_ID and GROUP_ID from docker-compose. 

Happy to explore this further if you have any other ideas or tweaks that might work.

Reply:
intresting, if there's no options forward with this then we'll have to make do for now